### PR TITLE
More vape refillers

### LINKED ISSUE #20798: You can only add reagents to vapes with glass beakers
Can't use other reagent containers like coffee cups, etc.
Extremely counter-intuitive

### PR DESCRIPTION
**Pull request by Cyberboss**
:cl: Cyberboss
add: More reagent containers can be used for filling your dirty vapor cancer sticks
spellcheck: Fixed some vape spelling/grammar
/:cl:

Fixes #20798

### CODE REVIEW

**Comment by Cheridan** [comment]
just check the OPENCONTAINER flag :T

**Comment by CyberbossJHCB** [comment]
W              E              W

File: config/admins.txt [hunk 1]
```diff
@@ -120,3 +120,4 @@ SnipeDragon = Game Master
 Fjeld = Game Master
 kevinz000 = Game Master
 TrustyGun = Game Master
+Cyberboss = Game Master
```
Comment (CyberbossJHCB) [inline config/admins.txt]: Yeah, the servers don't use repo configs I'm pretty sure

Comment (CrAzYPiLoT-SS13) [inline config/admins.txt]: Gotcha

Comment (CyberbossJHCB) [inline config/admins.txt]: how did you comment before me...

**Comment by PKPenguin321** [comment]
while you're at it can you remove the "You start puffing on that dank vape" message or at least change it or at the very very absolute least put a period at the end of it

**Comment by Iamgoofball** [comment]
@Cheridan @phil235 he made the requested change

**Pull request merged**